src/domain/baraba.py

```diff
@@ -1,15 +1,44 @@
 import random
 from dataclasses import dataclass
 
+from src.app.console import Console
```

Comment:
Zakaj importaš `Console` v domeno? Če importaš `Console` v domeni potem ustvariš dependency na Consolo.
In zaradi tega dobiš circular dependency v consoli... To je stvar ki te jo hočem s tem projektom naučiti in to je najpomembnejša stvar v tem projektu da se jo dobro naučiš in je za začetnika najtežja!

Domenski prostor je prostor v katerem so naštete tovarne objekte ki lahko obstajajo tudi brez računalnika... Hero princess, badguy ti objekti lahko živijo v realnosti, takoj pa ko importaš v domeno consol pa ustvariš dependency in zdaj je domena vezana da živi v consoli, vendar če bova hotla igro prestaviti v android tega ne bova morala narediti ker je igra vezana na consolo...

To bi bilo fino da bi se midva dobila v živo da bi ti res dobro razložil, tega ne moram preko besedila ti vsega razložiti na zalost.

Reply:
Tist circular import "fix" sm vedu, da ni končna rešitev problema, nism imeu več ideje kako zrihtat, moram zdej jit vse čez...časa sem si dau en teden, da nardim celo delujočo igro, in vmes meu goro logičnih problemou in hroščev, btw to mi ni še nikoli ratalo nardit cele delujoče igre, tako da sm že to veseu😁 in na koncu mi je že mal vodo v grlo tekla zaradi časa...Se mi je pa zdelo, da bo treba veliko popravkov v strukturi in logiki  🙄 Ok, bom vse pogledal, in naštudiral...če mi kaj res ne bo jasno se slišiva.
Lp @urosjarc